src/test/java/net/sf/jabref/logic/importer/fetcher/MrDLibFetcherTest.java

```diff
@@ -0,0 +1,40 @@
+package net.sf.jabref.logic.importer.fetcher;
+
+import java.util.List;
+
+import net.sf.jabref.logic.importer.FetcherException;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.FieldName;
+import net.sf.jabref.testutils.category.FetcherTests;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import static org.junit.Assert.assertEquals;
+
+@Category(FetcherTests.class)
+public class MrDLibFetcherTest {
+
+    private MrDLibFetcher fetcher;
+    private BibEntry bibEntry;
+
+    @Before
+    public void setUp() {
+        fetcher = new MrDLibFetcher("", "");
+        bibEntry = new BibEntry();
+        bibEntry.setField(FieldName.TITLE, "lernen");
+    }
+
+    @Test
+    public void testPerformSearch() throws FetcherException {
+        List<BibEntry> bibEntrys = fetcher.performSearch(bibEntry);
+        assertEquals(false, bibEntrys.isEmpty());
```

Comment:
Sorry, I don't see any new tests for the fetcher. Can you point/link me to them?
It is ok if you modify your entries on the server side and thus the test might fail in the future. It is still better to have a test that might need modifications in the future, instead of having no test at all.